config.js

```diff
@@ -341,7 +341,9 @@ config.iota = {
     /**
      * Default type, for IoT Agent installations that won't require preregistration.
      */
-    defaultType: 'Thing'
+    defaultType: 'Thing',
+
+    explicitAttrs: false
```

Comment:
```suggestion
    /**
     * flag indicating whether the incoming measures to the IoTAgent should be processed as per the "attributes" field.
     */
    explicitAttrs: false
```

To mimics how looks like in the sibling PR at IOTA-JSON